filters/advanced-bloom/src/extract-brightness.frag

```diff
@@ -13,9 +13,9 @@ void main() {
     float _min = min(min(color.r, color.g), color.b);
     float brightness = (_max + _min) * 0.5;
 
-    if(brightness < threshold) {
-        gl_FragColor = vec4(0.0, 0.0, 0.0, 0.0);
-    } else {
+    if(brightness > threshold) {
```

Comment:
Isn't this saying the same thing because you changed `<` to `>` and reversed the `else`?

Reply:
Fixed. thank you.

Reply:
Yes.  I typo in slack.  the current code is right. 
My real mean is that  :